frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -108,6 +110,27 @@ const asVMSettingsFieldValidator = (
     subject: getFieldTitle(iGetFieldKey(field)),
   });
 
+const validateSource: Validator = (field, options): ValidationObject => {
+  const value = iGetFieldValue(field);
+
+  if (value === ProvisionSource.PXE.getValue()) {
+    const { getState, id } = options;
+    const state = getState();
+    const isCreateTemplate = iGetCommonData(state, id, VMWizardProps.isCreateTemplate);
+    if (isCreateTemplate) {
+      return null;
+    }
+
+    const nads = iGetCommonData(state, id, VMWizardProps.nads);
+    const missingNADs = iGetLoadError(nads) || iGetLoadedData(nads)?.size === 0;
+    if (missingNADs) {
+      return asValidationObject(SELECT_PXE_NAD_ERROR_INFO);
```

Comment:
can we use the similar messages and logic as in the nic modal? A user choosing a PXE will not be able to navigate to the nic modal once s/he sees this error. And will not get get the more detailed message.

Reply:
sure